README.md

```diff
@@ -1,19 +1,30 @@
 ![Snyk logo](https://snyk.io/style/asset/logo/snyk-print.svg)
 
-***
+---
+
 # Snyk cloud-config-parser
 
-A utility library for identifying the issue path in a YAML/JSON/HCL file and returning the relevant line number in order to highlight the relevant line to the users in their files.
+A utility library that support Snyk IaC parsing.
+The two supported types of functionalities are:
+
+1. Identify Issue Path in file.
```

Comment:
```suggestion
1. Identify issue path in a cloud config file.
```

Reply:
Reading this, I do actually think there are good reasons for both of these things to live in the same repo, they're very closely coupled.